content/articles/2018-10-ggplot2-3-1-0.Rmarkdown

```diff
@@ -0,0 +1,159 @@
+---
+title: 'ggplot2 3.1.0'
+author: Mara Averick and Claus Wilke
+date: '2018-10-26'
+slug: ggplot2-3-1-0
+description: > 
+  ggplot2 3.1.0 is now on CRAN!
+categories:
+  - packages
+photo:
+  url: https://unsplash.com/photos/Ibq4B5iE_-4
+  author: Stephan Henning
+---
+
+```{r setup, include = FALSE}
+library(ggplot2)
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+We're happy to announce the release of
+[ggplot2 3.1.0](https://ggplot2.tidyverse.org/) on CRAN. For a complete list of
+changes, please see the [release notes](https://github.com/tidyverse/ggplot2/releases/tag/v3.1.0).
+
+## Breaking changes
+
+This is a minor release and breaking changes have been kept to a minimum. End users of ggplot2 are unlikely to encounter any issues. However, there are a few items that developers of ggplot2 extensions should be aware of.[^deets]
```

Comment:
Footnotes kinda suck with hugo's markdown, so I think it would be better as a parenthetical.

Generally this seems like a bit too much focus on backward incompatible changes — could we just reduce to a sentence each and link to the NEWS for full details?